lib/src/arithmetic/values/fixed_point_value.dart

```diff
@@ -16,7 +16,7 @@ export 'fixed_point_populator.dart';
 
 /// An immutable representation of (un)signed fixed-point values following
 /// Q notation (Qm.n format) as introduced by
-/// (Texas Instruments)[https://www.ti.com/lit/ug/spru565b/spru565b.pdf].
+/// (Texas Instruments: (https://www.ti.com/lit/ug/spru565b/spru565b.pdf).
```

Comment:
typo: this file and `fixed_point_logic.dart` have resolved this same lint issue in two different ways, one is missing a space and this one has unmatched parentheses